protocol/0090-VAMM-automated_market_maker.md

```diff
@@ -127,6 +127,18 @@ In addition to amending to reduce the size a user may also cancel their AMM enti
 
 Note that, whilst an `Abandon Position` transaction immediately closes the AMM a `Reduce-Only` transaction will keep it active for an uncertain amount of time into the future. During this time, any Amendment transactions received should move the AMM out of `Reduce-Only` mode and back into standard continuous operation.
 
+### Oracle-driven base price
+
+The base price can either be a fixed scalar specified by a user or an [oracle configuration](./0045-DSRC-data_sourcing.md). It should be possible to switch between the two with a regular vAMM amendment transaction.
+
+When oracle-driven base price is selected during the vAMM creation then the system first awaits the receipt of the next data point from the oracle and then follows the process outlined in [creation/amendment process](#creationamendment-process) with `base price` set to that value.
```

Comment:
Maybe a word on what to do about slippage? When you submit an AMM the transaction comes with a slippage field that controls how far to to allow the AMM's fair-price can move from its base-price when it tries to "rebase" the AMM so that it doesn't cross with whats already on the book. An amend transaction has one too which works the same but between the amended AMM's quote price and where it would need to be to not cross.

Do we want something similar here too? For example if the oracle-drive price is wildly different from where the AMM is currently at? Or should it always try to rebase itself even if it means eat up a lot of volume to get there?

Reply:
Should use the slippage specified when the vAMM was first set up, if this is an amendment that specifies slippage then the new slippage should be used.

Reply:
Modified the spec to use the slippage